src/components/calendar/Calendar.tsx

```diff
@@ -292,8 +327,38 @@ export default function Calendar() {
   ]);
 
   useEffect(() => {
-    setFilter(generateScheduledEventFilter(queryInstrumentId, startsAt, view));
-  }, [queryInstrumentId, startsAt, view]);
+    if (
```

Comment:
Maybe logic in if could be extracted to a variable with descriptive name

Reply:
Good point!